docs/source/en/model_doc/bort.md

```diff
@@ -16,6 +16,12 @@ rendered properly in your Markdown viewer.
 
 # BORT
 
+<Tip warning={true}>
+
+This model is in maintenance mode only, so we won't accept any new PRs changing its code.
```

Comment:
I wonder if we shouldn't also mention the last version of `transformers` in which this model was supported. So something like this:

```suggestion
This model is in maintenance mode only, so we won't accept any new PRs changing its code.

If you run into any issues running this model, please reinstall the last version that supported this model: v4.30.0. You can do so by running the following command: `pip install -U transformers==4.30.0`.
```